### PR TITLE
Allow setting different log levels for log tags

### PR DESCRIPTION
**Pull request by a-hassanin**
This feature allows using logging.tags in the configuration to set different log levels, and thus overwriting the already set logger log_level but only for the specified tag. Each log message is tagged, and if the used tag has a different log level than the logger, the tag log level will be used.

An interesting use case for which this is implemented in particular is the ForwardHandler log tag, in which it includes recursion messages which are needed to analyse dns traffic.  To view these messages currently, it is only possible using DEBUG level which is not an option in production landscapes as it might overload the logging platform with too many messages. Using this approach will allow the user to specify the intention of viewing DEBUG messages of the ForwardHandler tagged messages alone. An example of the configuration is shown in the below example:
 
```
log_level: ERROR # the property that already exists
logging:
- format: xxx
- tags:
  - name: ForwardHandler
    log_level: DEBUG
```

NOTE: 
- This is based on the agreement reached in the discussion of [this PR](https://github.com/cloudfoundry/bosh-dns-release/pull/78).
-  It depends on https://github.com/cloudfoundry/bosh-utils/pull/85

### CODE REVIEW

**Review by ystros — APPROVED** [review]
Looks good, once the related `bosh-utils` package PR is merged and we've bumped to it!

**Comment by rkoster** [comment]
@a-hassanin could you take a look at the failing tests?

**Comment by a-hassanin** [comment]
> @a-hassanin could you take a look at the failing tests?

It needs to be bumped to use bosh-utils [v0.0.425](https://github.com/cloudfoundry/bosh-utils/releases/tag/v0.0.425)

File: jobs/bosh-dns/spec [hunk 1]
```diff
@@ -174,3 +174,6 @@ properties:
   logging.format.timestamp:
     description: "Format for the timestamp in the component logs.  Valid values are 'rfc3339' and 'deprecated'."
     default: "rfc3339"
+
+  logging.tags:
+      description: "Tags for logging to provide individual configuration to the logger."
```
Comment (beyhan) [inline jobs/bosh-dns/spec]: Can we state here that this is optional and what happens if not configured. This part `to provide individual configuration to the logger.` says a little bit what will happen but it is not completely clear.

**Review by beyhan — COMMENTED** [review]
I also think that most probably makes sense to document this in https://bosh.io/docs/dns/.

**Review by beyhan — APPROVED** [review]

**Pull request merged**